src/utils.js

```diff
@@ -1,12 +1,10 @@
-/*import * as fs from "fs";
+import * as fs from "fs";
```

Comment:
is the fs module useful here? looks like it's only used in `index.js`

Reply:
it's used to create a write stream in L4 (writing to a file)